src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
From a user experience perspective, I'm a little confused about the purpose of "clear". Is the intention that I type `/wfh clear` and that deletes my WFH event for the current day?